hibikase/LineTimingDecorations.cpp

```diff
@@ -65,6 +61,33 @@ SyllableDecorations::SyllableDecorations(const QPlainTextEdit* text_edit, int st
     setVisible(true);
 }
 
+QColor SyllableDecorations::GetPlayingColor() const
+{
+    return QPalette().color(QPalette::Text);
+}
+
+QColor SyllableDecorations::GetNotPlayedColor() const
+{
+    QColor col(0x55, 0x77, 0x55);
+    // If the system text colour is darker than our colours, make them lighter
+    // (Assumes a system with conventional grayscale text fg/bg colours…)
+    int diff = QPalette().color(QPalette::Text).lightness() - col.lightness();
+    if (diff > 0) {
```

Comment:
We put the brace on the next line, not the current line. Or since there's only one line inside, you can do this:

```
if (diff > 0)
    col = col.lighter(diff);
```